### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by tanmaypardeshi**
I have updated the README.md contents for the links and some other things. I have also left some comments where I am not sure what to. Please let me know your suggestions in the review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,46 +51,52 @@ For 3.0, we created new tools for instructors and students to use to improve cou
...
+<!-- Need to update this -->
 [Click here to see the features of iterations I and II.](https://github.com/chandur626/TeachersPetBot/blob/update-readme/docs/feature-history.md)
 
+<!-- Need to update this -->
 <h4>Charts</h4>
 Instructors (like TAs, and Professors) can quickly make graphcs and charts directly in discord to share with students/users. Instructors can use this feature to share grade distributions, lecture participation/attendance, or other course statistics. All charts are named and stored into a json file when they are created. Students have acess to a command that allows them to view previously presented charts.
 
 <p align="left"><img width=65% src="https://github.com/chandur626/TeachersPetBot/blob/update-readme/docs/media/charts.gif"></p>
 
+<!-- Need to update this -->
```
Comment (tanmaypardeshi) [inline README.md:63]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 2]
```diff
@@ -49,46 +51,52 @@ For 3.0, we created new tools for instructors and students to use to improve cou
...
 <h4>Email Configuration</h4>
 
 This feature enables users to configure their email address in the system to receive important notifications, attachments from professors, assignment reminders. Users can also update, view and unconfigure a configured email address through the system.
 
 <p align="left"><img width=65% src="https://github.com/chandur626/TeachersPetBot/blob/update-readme/docs/media/email-address.gif"></p>
 
+<!-- Need to update this -->
```
Comment (tanmaypardeshi) [inline README.md:70]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 3]
```diff
@@ -49,46 +51,52 @@ For 3.0, we created new tools for instructors and students to use to improve cou
...
 <h4>Email Interaction</h4>
 
 This feature notifies all students regarding the next assignment deadline which is due for a day through email.
 
 <p align="left"><img width=65% src="https://github.com/chandur626/TeachersPetBot/blob/update-readme/docs/media/email-reminder.gif"></p>
 
+<!-- Need to update this -->
```
Comment (tanmaypardeshi) [inline README.md:77]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 4]
```diff
@@ -49,46 +51,52 @@ For 3.0, we created new tools for instructors and students to use to improve cou
...
 <h4>Re-Grading</h4>
 
 This feature provides a way for students to submit regrade requests and Instructors can collect information of the requests submitted. There are various commands included to add, update, display and remove regrade requests.
 This usecase was based on regrade request submission for CSE 510 SE FALL21 mid examination.
 
 <p align="left"><img width=65% src="https://github.com/chandur626/TeachersPetBot/blob/update-readme/docs/media/Regrade.gif"></p>
 
-
+<!-- Need to update this -->
```
Comment (tanmaypardeshi) [inline README.md:85]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 5]
```diff
@@ -49,46 +51,52 @@ For 3.0, we created new tools for instructors and students to use to improve cou
...
 <h4>Link Saving</h4>
 
 This feature is helpful to save all the messages which contain important URLs. we have built a user command "!send_links"  This command lets users access all messages which contain URLs. The messages Containing URLs are automatically get appended in a file and the file is attached when the "!send_links" command is input.
 
 <p align="left"><img width=65% src="https://github.com/chandur626/TeachersPetBot/blob/update-readme/docs/media/link-saving.gif"></p>
 
+<!-- Need to update this -->
```
Comment (tanmaypardeshi) [inline README.md:92]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 6]
```diff
@@ -49,46 +51,52 @@ For 3.0, we created new tools for instructors and students to use to improve cou
...
 <h4>Project Event</h4>
 
 This feature allows instructors or teaching assistants to create a project event by providing information such as description, link for project submission and deadline. The deadline reminder is taken care of Email Interaction feature.
 
 <p align="left"><img width=65% src="https://github.com/chandur626/TeachersPetBot/blob/update-readme/docs/media/project-event.gif"></p>
 
-
+<!-- Need to update this -->
```
Comment (tanmaypardeshi) [inline README.md:99]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 7]
```diff
@@ -100,21 +108,22 @@ This feature is used to detect spam in message channels. When a user tries to se
 <h2> Installation and Running </h2>
 
 #### Tools and Libraries Used
-In addition to the packages from [requirements.txt](https://github.com/Ashwinshankar98/TeachersPetBot/blob/main/requirements.txt) which need to be installed, please have the following installed on your machine:
+In addition to the packages from [requirements.txt](https://github.com/tanmaypardeshi/CSC-510-Project2-TeachersPetBot/blob/main/requirements.txt) which need to be installed, please have the following installed on your machine:
+<!-- Are there any changes here? -->
```
Comment (tanmaypardeshi) [inline README.md:112]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 8]
```diff
@@ -100,21 +108,22 @@ This feature is used to detect spam in message channels. When a user tries to se
...
 * [Python 3.9.7](https://www.python.org/downloads/)
 * [Sqlite](https://www.sqlite.org/download.html)
-
+<!-- Are there any changes here? -->
```
Comment (tanmaypardeshi) [inline README.md:115]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 9]
```diff
@@ -100,21 +108,22 @@ This feature is used to detect spam in message channels. When a user tries to se
...
 To install and run Teacher's Pet, follow instructions in the [Installation and Testing Guide](https://github.com/Ashwinshankar98/TeachersPetBot/blob/main/Installation.md).
 
 
 <a name="testing"></a>
-
+<!-- Are there any changes here? -->
```
Comment (tanmaypardeshi) [inline README.md:120]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md [hunk 10]
```diff
@@ -100,21 +108,22 @@ This feature is used to detect spam in message channels. When a user tries to se
...
 <h2>Testing </h2>
 
 To run tests on the Teacher's Pet, follow instructions in the [Installation and Testing Guide](https://github.com/Ashwinshankar98/TeachersPetBot/blob/main/Installation.md#Run-Tests).
 
 <hr />
-
+<!-- Are there any changes here? -->
```
Comment (tanmaypardeshi) [inline README.md:126]: @ExtremeMachine12 Please let me know what to do here if you know.

File: README.md (hunk 8 above)
Comment (ExtremeMachine12) [reply README.md:115]: We can update this to Python 3.10

**Review by ExtremeMachine12 — DISMISSED** [review]
Looks good! We will need to change some of the links and gifs as we finish the extra features.

**Review by ExtremeMachine12 — APPROVED** [review]
Awesome job!

**Pull request merged**